### PR TITLE
[MM-60856] Replace FormattedMarkdownMessage in 'webapp/channels/src/components/linking_landing_page/linking_landing_page.tsx' with FormattedMessage

### LINKED ISSUE #28698: Replace FormattedMarkdownMessage in 'webapp/channels/src/components/linking_landing_page/linking_landing_page.tsx' with FormattedMessage
The goal of this task is to remove all instances of “FormattedMarkdownMessage” (imported from “components/formatted_markdown_message”) in the file `webapp/channels/src/components/linking_landing_page/linking_landing_page.tsx`. These should be replaced with the “FormattedMessage” component from the “react-intl” library.

Ensure that rich text formatting is preserved by following the implementation guidelines in the react-intl (https://formatjs.io/docs/react-intl/components#rich-text-formatting) documentation. Make sure to change the message ID after the update as the current extraction tool does not automatically detect changes in the messages. Once done, make sure to run `npm run i18n-extract` in the “webapp/channels” directory to pick up the new messages. For guidance refer to the example pull request - https://github.com/mattermost/mattermost/pull/28438

----
If you're interested please comment here and come [join our "Contributors" community channel](https://community.mattermost.com/core/channels/tickets) on our daily build server, where you can discuss questions with community members and the Mattermost core team. For technical advice or questions, please  [join our "Developers" community channel](https://community.mattermost.com/core/channels/developers).

New contributors please see our [Developer's Guide](https://developers.mattermost.com/contribute/getting-started/).

JIRA: https://mattermost.atlassian.net/browse/MM-60856

### PR DESCRIPTION
**Pull request by RS-labhub**
#### Summary
Replaced FormattedMarkdownMessage in '../linking_landing_page/linking_landing_page.tsx' with FormattedMessage

#### Ticket Link

Fixes #28698 
Jira [MM-60856](https://mattermost.atlassian.net/browse/MM-60856)

#### Release Note

```release-note
NONE
```

[MM-60856]: https://mattermost.atlassian.net/browse/MM-60856?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: webapp/channels/src/components/linking_landing_page/linking_landing_page.tsx [hunk 1]
```diff
@@ -241,11 +241,18 @@ export default class LinkingLandingPage extends PureComponent<Props, State> {
...
-                        defaultMessage='Or, [open this link in your browser.](!{link})'
+                    <FormattedMessage
+                        id='getApp.downloadLinkInBrowser'
+                        defaultMessage='Or, <a>open this link in your browser</a>.'
                         values={{
-                            link: this.state.location,
+                            a: (chunks: string) => (
+                                <ExternalLink
+                                    href={this.state.location}
+                                    location='get_app'
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/linking_landing_page/linking_landing_page.tsx:251]: Lets change this location to `landingPage`

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

File: webapp/channels/src/components/linking_landing_page/linking_landing_page.tsx (hunk 1 above)
Comment (RS-labhub) [reply webapp/channels/src/components/linking_landing_page/linking_landing_page.tsx:251]: Done!

**Review by RS-labhub — COMMENTED** [review]
Done!

**Review by M-ZubairAhmed — APPROVED** [review]
This looks good :shipit: Thank you

**Review by larkox — APPROVED** [review]

**Review by yasserfaraazkhan — APPROVED** [review]

**Pull request merged**